MetaMask/metamask-extension
# I18n: Fixed and Added Korean translation.

## **Description**
Fixed and Added Korean translation.

## **Related issues**
No

## **Manual testing steps**

1. yarn install && yarn start
2. load extension
3. check in korean browser.

## **Screenshots/Recordings**

### **Before**
#### main
<img width="358" alt="main-before" src="https://github.com/MetaMask/metamask-extension/assets/42014299/a4763416-639a-486e-abea-c136cac69017">

#### onboarding
<img width="620" alt="onboarding-before" src="https://github.com/MetaMask/metamask-extension/assets/42014299/2f011bc8-5661-4ecc-be19-5a46ef183964">

#### add account
<img width="363" alt="add-before" src="https://github.com/MetaMask/metamask-extension/assets/42014299/518ac8ab-260a-48d1-9dbb-48b9b1045c75">

#### snpas
<img width="278" alt="snaps-before" src="https://github.com/MetaMask/metamask-extension/assets/42014299/2e2f46e7-e6cb-446c-a8cc-f1d8aa1d40ce">

#### nft
<img width="367" alt="nft-auto-before" src="https://github.com/MetaMask/metamask-extension/assets/42014299/08756bcb-064d-41e5-932e-08ef768d767d">

### **After**
#### main
<img width="356" alt="main-after" src="https://github.com/MetaMask/metamask-extension/assets/42014299/e0df86b4-57b9-4741-90c3-e0726c606739">


#### onboarding
<img width="625" alt="onboarding-after" src="https://github.com/MetaMask/metamask-extension/assets/42014299/e051b986-d092-4442-8b62-6c9e57e27428">


#### add account
<img width="347" alt="add-after" src="https://github.com/MetaMask/metamask-extension/assets/42014299/eabbbddd-3215-4eb8-b057-d437ad90f1b0">

#### snpas
<img width="357" alt="snaps-after" src="https://github.com/MetaMask/metamask-extension/assets/42014299/d69c2ed2-26ce-4371-afc4-ef889470a023">

#### nft
<img width="366" alt="nft-auto-after" src="https://github.com/MetaMask/metamask-extension/assets/42014299/e670f604-243a-4926-bc8e-047723242c13">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
